checkstyle.properties

```diff
@@ -1,15 +1,23 @@
 category=External Analysers
 description=Provide Checkstyle rules for Java projects
 homepageUrl=https://github.com/checkstyle/sonar-checkstyle
-archivedVersions=10.5,10.6.0,10.7.0,10.8.1,10.9.3,10.11.0,10.12.0,10.12.1,10.12.3,10.12.5,10.14.1,10.14.2,10.15.0,10.16.0,10.17.0,10.19.0,10.20.1
-publicVersions=10.20.2
+archivedVersions=10.5,10.6.0,10.7.0,10.8.1,10.9.3,10.11.0,10.12.0,10.12.1,10.12.3,10.12.5,10.14.1,10.14.2,10.15.0,10.16.0,10.17.0,10.19.0,10.20.1,10.20.2
+publicVersions=10.21.1
 
 defaults.mavenGroupId=org.sonarsource.checkstyle
 defaults.mavenArtifactId=sonar-checkstyle-plugin
 
+10.21.1.description=Upgrade to Checkstyle 10.21.1
+10.21.1.sqs=[10.8,LATEST]
+10.21.1.sqcb=[24.12,LATEST]
+10.21.1.sqVersions=[9.9,10.7.*]
+10.21.1.date=2025-01-09
+10.21.1.changelogUrl=https://github.com/checkstyle/sonar-checkstyle/releases/tag/10.21.1
+10.21.1.downloadUrl=https://github.com/checkstyle/sonar-checkstyle/releases/download/10.21.1/checkstyle-sonar-plugin-10.21.1.jar
+
 10.20.2.description=Upgrade to Checkstyle 10.20.2
-10.20.2.sqs=[10.8,LATEST]
-10.20.2.sqcb=[24.12,LATEST]
+10.20.2.sqs=[10.8,10.8.*]
+10.20.2.sqcb=[24.12,24.12.*]
```

Comment:
25.1 was just released. Please end this range there.

Reply:
thank you, i updated the range. 😃